settings.gradle.kts

```diff
@@ -8,6 +8,10 @@ plugins {
 
 rootProject.name = "gravitino"
 
+val scalaVersion: String = gradle.startParameter.projectProperties["scalaVersion"]?.toString()
+  ?: settings.extra["defaultScalaVersion"]?.toString()
+  ?: "2.12"
```

Comment:
Why not just define `defaultScalaVersion` directly in the `settings` file, it seems unnecessary to hardcode a `2.12` here.

Reply:
`defaultScalaVersion` is defined in `gradle.properties`, removed `2.12`